api/swagger-spec/oapi-v1.json

```diff
@@ -16987,6 +16987,10 @@
        "$ref": "v1.DeploymentCause"
       },
       "description": "extended data associated with all the causes for creating a new deployment"
+     },
```

Comment:
The field it refers to is actually `status.details.message`, so it seems misleading in two ways: it's not the time of the status field itself was updated, and `status.details.causes` is updated independently elsewhere. Kind of a mess IMO. Thoughts?